.github/workflows/gitlab-ci.yaml

```
@@ -144,15 +144,15 @@ jobs:
       - name: Run migrations
         run: |
           python manage.py migrate
-
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)